pinot-segment-local/src/main/java/org/apache/pinot/segment/local/function/InbuiltFunctionEvaluator.java

```diff
@@ -69,6 +69,8 @@ private ExecutableNode planExecution(ExpressionContext expression) {
           childNodes[i] = planExecution(arguments.get(i));
         }
         String functionName = function.getFunctionName();
+        Preconditions.checkState(FunctionRegistry.containsFunction(functionName), "Unsupported function: %s not found", 
```

Comment:
Same here

Reply:
Addressed in https://github.com/apache/pinot/commit/f02a015291428e62f7f3dc31b10a323844a4e447